{"id": 592386, "name": "Manmeet Kaur", "username": "21f3001993", "avatar_template": "/user_avatar/discourse.onlinedegree.iitm.ac.in/21f3001993/{size}/59073_2.png", "created_at": "2025-02-09T06:34:11.908Z", "cooked": "<p>I posted this error message but now the first issue got resolved but I am still keeping it in my post so that if anyone faces same issue, they can try if they can fix it similar to how it worked for me.</p>\n<p>Please help with the second issue.</p>\n<ol>\n<li>For Q8, the workflow is running on Github and commiting the scraped results to the json file (which is so amazing for me btw!). But I am getting this error for my public repo.<br>\nHow it got resolved: I set up fixed time for cron schedule instead of every 5 min. Now it works.</li>\n</ol>\n<blockquote>\n<p>Error: No daily scheduled triggers found in workflows.</p>\n</blockquote>\n<ol start=\"2\">\n<li>I had all correct results for Q1 to Q7 till yesterday but it keeps giving errors even when I reload same file for some questions. Do I need to keep addressing those errors or if once correct and saved, I can ignore those?</li>\n</ol>", "post_number": 205, "post_type": 1, "posts_count": 370, "updated_at": "2025-02-09T07:43:12.369Z", "reply_count": 0, "reply_to_post_number": null, "quote_count": 0, "incoming_link_count": 0, "reads": 59, "readers_count": 58, "score": 11.6, "yours": false, "topic_id": 165959, "topic_slug": "ga4-data-sourcing-discussion-thread-tds-jan-2025", "display_username": "Manmeet Kaur", "primary_group_name": "ds-students", "flair_name": "ds-students", "flair_url": null, "flair_bg_color": "", "flair_color": "", "flair_group_id": 294, "badges_granted": [], "version": 3, "can_edit": false, "can_delete": false, "can_recover": false, "can_see_hidden_post": false, "can_wiki": false, "read": true, "user_title": null, "bookmarked": false, "actions_summary": [{"id": 2, "can_act": true}, {"id": 6, "can_act": true}, {"id": 3, "can_act": true}, {"id": 4, "can_act": true}, {"id": 8, "can_act": true}, {"id": 10, "can_act": true}, {"id": 7, "can_act": true}], "moderator": false, "admin": false, "staff": false, "user_id": 8102, "hidden": false, "trust_level": 1, "deleted_at": null, "user_deleted": false, "edit_reason": null, "can_view_edit_history": true, "wiki": false, "post_url": "/t/ga4-data-sourcing-discussion-thread-tds-jan-2025/165959/205", "user_cakedate": "2021-12-29", "reactions": [], "current_user_reaction": null, "reaction_users_count": 0, "current_user_used_main_reaction": false, "can_accept_answer": false, "can_unaccept_answer": false, "accepted_answer": false, "topic_accepted_answer": null}